actioncable/lib/action_cable/connection/client_socket.rb

```diff
@@ -29,7 +29,7 @@ def self.secure_request?(env)
 
       attr_reader :env, :url
 
-      def initialize(env, event_target, event_loop)
+      def initialize(env, event_target, event_loop, protocols: ActionCable::INTERNAL[:protocols])
```

Comment:
ditto -- I think we can drop the default here for protocols -- the option to pass in a different array is already presented to users in Connection::WebSocket#initialize